gnucash/import-export/import-main-matcher.c

```diff
@@ -189,8 +200,11 @@ on_matcher_ok_clicked (GtkButton *button, GNCImportMainMatcher *info)
     /*   DEBUG ("Begin") */
 
     model = gtk_tree_view_get_model (info->view);
-    if (!gtk_tree_model_get_iter_first (model, &iter))
+    if (!gtk_tree_model_get_iter_first (model, &iter)) {
```

Comment:
Please put the `{` on its own line.  See the [GnuCash Coding Style](https://wiki.gnucash.org/wiki/CodingStandard).